teaser/examples/e1_generate_archetype.py

```diff
@@ -95,24 +95,34 @@ def example_generate_archetype():
     # data. Project(load_data=False).
 
     prj.add_residential(
-        method='tabula_de',
-        usage='single_family_house',
+        construction_data='tabula_de_standard',
+        geometry_data='tabula_de_single_family_house',
         name="ResidentialBuildingTabula",
         year_of_construction=1988,
         number_of_floors=3,
         height_of_floors=3.2,
-        net_leased_area=280.0,
-        construction_type='tabula_standard')
+        net_leased_area=280.0)
 
     prj.add_residential(
-        method='tabula_de',
-        usage='multi_family_house',
+        construction_data='tabula_de_retrofit',
+        geometry_data='tabula_de_multi_family_house',
         name="ResidentialBuildingTabulaMulti",
         year_of_construction=1960,
         number_of_floors=4,
         height_of_floors=3.2,
-        net_leased_area=600.0,
-        construction_type='tabula_retrofit')
+        net_leased_area=600.0)
+
+    # This part is still commented out, otherwise example7 (retrofit) will not work
+
+    #prj.add_residential(
+    #    construction_data='kfw_40',
+    #    geometry_data='iwu_single_family_dwelling',
+    #    name="ResidentialBuildingKfw",
+    #    year_of_construction=2024,
+    #    number_of_floors=2,
+    #    height_of_floors=3.2,
+    #    net_leased_area=200.0
+    #)
```

Comment:
Should work, or? If not, remove it.

Reply:
The retrofit method does not yet work with the kfw standards, so I will remove it from the example.